qgis4/ansible/requirements.txt

```diff
@@ -0,0 +1 @@
+ansible==2.4.2.0
```

Comment:
How about using the latest ansible version (2.7 IIRC)?

Reply:
I took an old requirements file, I'll try to re-launch the deployment process with Ansible 2.7.

Reply:
It should be compatible.